data/resources/ui/components-message-entry.ui

```diff
@@ -30,21 +30,15 @@
                 <property name="hexpand">True</property>
                 <property name="child">
                   <object class="GtkTextView" id="text_view">
-                    <property name="top-margin">6</property>
-                    <property name="bottom-margin">6</property>
-                    <property name="left-margin">9</property>
-                    <property name="right-margin">9</property>
+                    <property name="top-margin">8</property>
```

Comment:
Without margin-bottom, this is the result of the text view when is scrolled to the bottom:

![image](https://user-images.githubusercontent.com/23530586/200896906-c7e35f51-2ae1-442b-8836-6bd0d9e08fd2.png)

Also, 6px margin is the correct value to keep it similar to the text margin in the message bubbles.